File: dp16.js
// # Write a while loop that prints the phrase "Around the world" 144 times.

var word = "around the world"
 
var count = 0

while (count < 144) {
  console.log(word)
  count = count + 1
}

// # Write a while loop that asks the user to enter a word and will run forever until the user enters a word with more than 5 letters.

while (true) {
  var user_input = window.prompt("enter a word:");
  if (user_input.length > 5) {
    break;
  }
}

// while (true) {
//   console.log("enter a word")
//   var user_input = gets.chomp
//   if user_input > 5
//   break
// }

File: dp16.rb
# Write a while loop that prints the phrase "Around the world" 144 times.

index = 0

while index < 144
  pp "Around the world"
  index = index + 1
end

# Write a while loop that asks the user to enter a word and will run forever until the user enters a word with more than 5 letters.

while true
  p "enter a word"
  user_input = gets.chomp
  if user_input.length > 5
    break
  end
end

# p "enter a word"
# p user_input = []

# index = 0
# if user_input.length < 5
#   p user input[index]
#   index = index + 1
# elsif user_input > 5
# break
